constrain/demo/tspr_cases/tspr_cases.py

```diff
@@ -0,0 +1,166 @@
+"""The goal of this demo cases is to showcase how verification cases can be semi-automatically created for building energy simulation timeseries.
+"""
```

Comment:
Add new demo script that showcases simulation-based verification using the ConStrain APIs.